cfg.go

```diff
@@ -1,65 +1,35 @@
 package cfg
 
 import (
-	"flag"
-	"log"
-	"os"
-	"strings"
+	"fmt"
 
-	"github.com/joho/godotenv"
 	"github.com/spf13/viper"
 )
 
 const (
-	envModeVar      = "GOENV"
-	localConfigName = "local"
+	configType        = "yaml"
+	defaultConfigName = "default"
 )
 
-var (
-	configPath string
-	configType string
-)
-
-func loadEnvConfig(env string) error {
-	viper.SetConfigName(env)
-	viper.AddConfigPath(configPath)
-	return viper.MergeInConfig()
-}
-
-// Load loads the configuration file depending on the Go environment mode
-func Load(serviceName string) {
-	flag.StringVar(&configPath, "config", "./config", "Configuration dir path")
```

Comment:
Agree with removing dependency on flag but `./config` must be the default path in the pkg

Reply:
Working on this.